test/hotspot/jtreg/compiler/jeandle/fileCheck/TestFileCheck.java

```diff
@@ -39,7 +39,7 @@ public static void main(String[] args) throws Exception {
             FileCheck fileCheck = new FileCheck(currentDir,
                                                 TestFileCheck.class.getDeclaredMethod("add", int.class, int.class),
                                                 false);
-            fileCheck.check("define hotspotcc i32 @\"compiler_jeandle_fileCheck_TestFileCheck_add_(II)I\"(i32 %0, i32 %1) gc \"hotspotgc\" {");
+            fileCheck.check("define hotspotcc i32 @\"compiler_jeandle_fileCheck_TestFileCheck_add_(II)I\"(i32 %0, i32 %1) #3 gc \"hotspotgc\" {");
```

Comment:
```suggestion
            fileCheck.check("define hotspotcc i32 @\"compiler_jeandle_fileCheck_TestFileCheck_add_(II)I\"(i32 %0, i32 %1)");
```
We can simply don't check the following attribute.